api/src/certification/session/infrastructure/storage/cpf-receipts-storage.js

```diff
@@ -28,6 +28,10 @@ class CpfReceiptsStorage {
     return deserialize({ xmlStream: data.Body });
   }
 
+  async deleteReceipt({ cpfReceipt }) {
+    await this.#client.deleteFile({ key: cpfReceipt.filename });
+  }
```

Comment:
Le "await" est deja effectué par l'appelant, inutile de rajouter celui ci
```suggestion
 deleteReceipt({ cpfReceipt }) {
    return this.#client.deleteFile({ key: cpfReceipt.filename });
  }
```

Reply:
Bien vu